.github/workflows/dockerimage.yml

```diff
@@ -0,0 +1,14 @@
+name: Docker Image CI
```

Comment:
I don't know what this does.
If it has something to do with creating those docker images we talked about, maybe it will be useful.
If not, maybe someone will want to delete it.